src/main/kotlin/org/jetbrains/intellij/platform/gradle/resolvers/path/JavaRuntimePathResolver.kt

```diff
@@ -55,6 +56,7 @@ class JavaRuntimePathResolver(
          * The bundled JetBrains Runtime within the current IntelliJ Platform.
          */
         "JetBrains Runtime bundled within the IntelliJ Platform" to {
+            intellijPlatform.resolve()
```

Comment:
Without this the default configuration are not possible I think.

Reply:
Not necessary, and actually unwanted.